python/GangaLHCb/Lib/GaudiRunApp/GaudiRunApp.py

```diff
@@ -149,6 +150,25 @@ def prepare(self, force=False):
             self.unprepare()
             raise
 
+        self.cleanGangaTargetArea(this_build_target)
```

Comment:
It looks like `prepare()` is no longer returning a value. I guess you meant `return self.cleanGangaTargetArea(this_build_target)` or to put a `return 1` on the line after?